src/react-native/react-native-wizard.ts

```diff
@@ -104,6 +125,19 @@ export async function runReactNativeWizardWithTelemetry(
 
   const packageJson = await getPackageDotJson();
 
+  const isExpoManaged = isExpoManagedProject(packageJson);
```

Comment:
Suggestion: We should avoid using the read package.json, instead we should reread it, to avoid using an outdated reference of it when a write operation is done. 

This may not be happening here in this PR but I am adding a comment since i noticed the same packageJson being passed to multiple locations in this PR.

For context: https://github.com/getsentry/sentry-wizard/pull/493

Reply:
Yeah makes sense, I removed passing the package JSON and made it an explicit metro package name, as that's the only value needed.